dev/packages/alpha/cisco/0.0.2/kibana/visualization/80d0c1b0-498a-11e9-b8ce-ed898b5ef295.json

```diff
@@ -0,0 +1,126 @@
+{
+    "attributes": {
+        "description": "",
+        "kibanaSavedObjectMeta": {
+            "searchSourceJSON": {
+                "filter": [],
+                "query": {
+                    "language": "kuery",
+                    "query": ""
+                }
+            }
+        },
+        "savedSearchRefName": "search_0",
+        "title": "ASA Flows by Network Bytes [Filebeat Cisco]",
```

Comment:
This is one of the lastest spottings - we need to get rid of all references to: filebeat, metricbeat and module in Kibana dashboards. These things do not exist anymore.